backend/usage/migrations/0001_initial.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 4.2.1 on 2024-04-22 12:55
+
+import uuid
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = []
+
+    operations = [
+        migrations.CreateModel(
+            name="Usage",
+            fields=[
+                ("created_at", models.DateTimeField(auto_now_add=True)),
+                ("modified_at", models.DateTimeField(auto_now=True)),
+                (
+                    "id",
+                    models.UUIDField(
+                        default=uuid.uuid4,
+                        editable=False,
+                        primary_key=True,
+                        serialize=False,
+                    ),
+                ),
+                ("workflow_id", models.CharField(max_length=255)),
+                ("execution_id", models.CharField(max_length=255)),
+                ("adapter_instance_id", models.CharField(max_length=255)),
+                ("run_id", models.CharField(max_length=255)),
```

Comment:
@johnyrahul Better to rename this field to `prompt_run_id`.
cc @chandrasekharan-zipstack @Deepak-Kesavan @gaya3-zipstack @harini-venkataraman

Reply:
will handle this in PR where @gaya3-zipstack . Today this column is not getting used cc: @Deepak-Kesavan